### DISCUSSION TITLE
Docs i18n Support

### DISCUSSION THREAD

**Discussion by Lsnsh** [JanDeDobbeleer/oh-my-posh#4985] [Ideas]
May I ask if there are any plans to support documentation internationalization?

I noticed that the doc is built based on Docusaurus , and it may be necessary to reorganize the file structure and translate React and MD/MDX files.

https://docusaurus.io/docs/3.0.1/i18n/tutorial

**Comment by JanDeDobbeleer** [comment]
@Lsnsh No, I don't have that intention as I wouldn't be able to support that.

**Reply by Lsnsh** [reply]
Thank you for your reply, can understand.
